incubator/hnc/internal/validators/hierarchy.go

```diff
@@ -119,6 +119,15 @@ func (v *Hierarchy) handle(ctx context.Context, log logr.Logger, req *request) a
 		return allow("HNC SA")
 	}
 
+	if config.ExcludedNamespaces[req.hc.Namespace] {
+		reason := fmt.Sprintf("Cannot set the excluded namespace %q as a child of another namespace", req.hc.Namespace)
+		return deny(metav1.StatusReasonForbidden, reason)
+	}
+	if config.ExcludedNamespaces[req.hc.Spec.Parent] {
+		reason := fmt.Sprintf("Cannot set the parent to the excluded namespace %q", req.hc.Spec.Parent)
+		return deny(metav1.StatusReasonForbidden, reason)
+	}
+
```

Comment:
Should these cause conditions if the webhook is bypassed?

Reply:
Details added in https://github.com/kubernetes-sigs/multi-tenancy/issues/1443#issuecomment-811384761. I will mark this PR is part of #1443 instead of fixing it and work on it (standardize the condition) in a next PR.